solana/program_v2/packages/client/idl/src/solana_anchor_gateway.ts

```diff
@@ -261,6 +261,11 @@ export type SolanaAnchorGateway = {
           "name": "gatekeeperTokenAccount",
           "isMut": true,
           "isSigner": false
+        },
+        {
+          "name": "mint",
```

Comment:
In other existing interfaces we call this `mintAccount`. I would just align the naming (only `mint` is better I think).

Reply:
Was this a "bug" that we forgot to pass a mint into `gatekeeperWithdraw`?

Reply:
No bug, you don't need to pass the mint to the Transfer struct, but we changed it to TransferChecked for the 2022 support which required up to add it.

Reply:
>In other existing interfaces we call this mintAccount. I would just align the naming (only mint is better I think).
Yes! I will get this aligned thanks

Reply:
Thanks for the clarification!